### ISSUE TITLE
Integrate Travis CI with Gitter

### ISSUE THREAD

**Issue by ollelauribostrom** [ollelauribostrom/rebus#97]
Lets integrate Travis CI with gitter. Add the following to the `notifications` part in [.travis.yml](https://github.com/ollelauribostrom/rebus/blob/9a00c3cccd2f8f46bec99ab3283e2429a5391962/.travis.yml#L11)

```yml
  webhooks:
    urls:
      - https://webhooks.gitter.im/e/d6e8722e284fbf3d51b2
    on_success: change
    on_failure: always
```

**Comment by V1shvesh** [comment]
Hey @ollelauribostrom!!

**Comment by ollelauribostrom** [comment]
@V1shvesh Thanks but this seems to have been implemented in #99. If you feel like it, add add a new rebus to the project instead 🙂
